src/xhr.ts

```diff
@@ -41,6 +41,39 @@ export class JQueryRequestor extends Requestor {
   }
 }
 
+/**
+ * Uses fetch API to make Ajax requests
+ */
+export class FetchRequestor extends Requestor {
+  xhr<T>(settings: JQueryAjaxSettings): Promise<T> {
+    const url: string = settings.url || '';
+    let requestInit: RequestInit = {};
+    requestInit.method = settings.method;
+    
+    if (settings.data) {
+      requestInit.body = <string>settings.data;
```

Comment:
This always assumes you are doing a `POST`. What if someone wanted to use a HTTP `GET` ?

Reply:
If the method is `GET` the data is sent as query string now.